docker/testnet/docker-compose.yaml

```diff
@@ -99,6 +99,11 @@ services:
       - AMARU_NETWORK=testnet_42
     volumes:
       - a1-state:/srv/amaru
+    depends_on:
+      cardano-loader:
+        condition: service_completed_successfully
+      p1:
```

Comment:
Actually, I purposely don't want `amaru` to depend on a well-behaving cardano-node: It should start even if connections fail and keep trying.